src/Shop/ExternalCalculatorService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace DDDHH\Shop;
+
+use DDDHH\Hexagon\Hexagon;
+use DDDHH\Shop\Cart\Cart;
+
+class ExternalCalculatorService extends Hexagon
```

Comment:
About the `Hexagon` thing see my above mentions.

Generally I'd like to see tests. I know this tiny class here is mostly about integration but even this should be tested. If you'd use a `ExternalCalculatorServiceClient` interface you could easily test not only the integration but the `prepareParams` method. You could do those tests with your `Port` and `Adapter` interfaces also.